.gitmodules

```diff
@@ -12,8 +12,8 @@
 	url = https://github.com/pfalcon/berkeley-db-1.xx
 [submodule "lib/stm32lib"]
 	path = lib/stm32lib
-	url = https://github.com/micropython/stm32lib
-	branch = work-F4-1.13.1+F7-1.5.0+L4-1.3.0
+	url = https://github.com/AEMICS/stm32lib
+	branch = vendor
```

Comment:
This needs to be undone.

Reply:
Done, I added the branch with G4:
work-F0-1.9.0+F4-1.16.0+F7-1.7.0+G4-1.3.0+H7-1.6.0+L0-1.11.2+L4-1.17.0+WB-1.10.0+WL-1.1.0